README.md

```diff
@@ -154,6 +154,14 @@ then 4 times in a row and issue an appropriate alert.
 
 ## Running a test environment
 
+### Kwok Docker Container
```

Comment:
Why build it from scratch when we can use a pre-built image?
https://kwok.sigs.k8s.io/docs/user/all-in-one-image

Reply:
Done